packages/prio3/src/prio3.ts

```diff
@@ -27,70 +27,93 @@ interface DecodedShare {
   inputShare: bigint[];
   proofShare: bigint[];
   blind: Buffer | null;
-  hint: Buffer | null;
+  jointRandPartsHint: Buffer[] | null;
 }
 
 interface Share {
   blind: Buffer;
-  hint: Buffer;
+  jointRandPart: Uint8Array;
   inputShare: bigint[];
   proofShare: bigint[];
   inputShareSeed?: Buffer;
   proofShareSeed?: Buffer;
+  jointRandPartsHint?: Uint8Array[];
```

Comment:
I'm curious about the use of `Uint8Array` here instead of `Buffer`. In general, I think we should move towards exclusively using `Uint8Array` everywhere (or generically `ArrayBuffer`), but in the current code it seems more consistent to use `Buffer`s here and for `jointRandPart`. Is there a reason this should be different from `blind`, `inputShareSeed`, and `proofShareSeed`?